### PR TITLE
Add post about mocks and contracts

### PR DESCRIPTION
**Pull request by ConnorRigby**
I thought i'd write up this quick post to refer users to since i seem to answer this question pretty frequently

### CODE REVIEW

File: content/post/2018-09-25-mocks-and-explicit-contracts-expansion.md [hunk 1]
```diff
@@ -0,0 +1,219 @@
+---
+title: An expansion on Mocks and Explicit Contracts
+date: 2018-09-25
+author: Frank Hunleth
```
Comment (fhunleth) [inline content/post/2018-09-25-mocks-and-explicit-contracts-expansion.md]: Want to put your name as author? :)

Comment (ConnorRigby) [reply content/post/2018-09-25-mocks-and-explicit-contracts-expansion.md]: My copy paste skills are sub par.

**Comment by ConnorRigby** [comment]
No rush. Let me know if you find/want to change anything.

**Comment by fhunleth** [comment]
Merged. Manually... Thanks!

**Pull request closed without merging**